src/layers/SelectableScatterplotLayer.js

```diff
@@ -1,14 +0,0 @@
-import { CompositeLayer, ScatterplotLayer } from 'deck.gl';
-
-import { overlayBaseProps } from './selection-utils';
-
-export default class SelectableScatterplotLayer extends CompositeLayer {
-  renderLayers() {
-    const props = overlayBaseProps(this.props);
```

Comment:
@keller-mark Should we be keeping this? I never fully understood why this existed, but I did use the `DataFilterExtension` here (in `overlayBaseProps`) which should in theory be faster for selection but I am not sure.  I was under the impression it had GPU acceleration.

Reply:
I think we're in the clear here since we are not actually doing any filtering (since we don't have the base-overlay paradigm).  Seems ok to me then!